### PR TITLE
Add table of contents to README

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,19 @@
 # Forklift
 
+ * [Vagrant Deployment](#vagrant-deployment)
+ * [Direct Deployment](#direct-deployment)
```
Comment (jlsherrill) [inline README.md]: I know this terminology has been around for a while, but could we rename this to 'Non-vagrant deployment'

File: README.md [hunk 2]
```diff
@@ -1,5 +1,19 @@
...
+ * [Nightly Install](#nightly-production-install)
+ * [Bats Testing](#bats-testing)
+ * [Development Deployment](#development-deployment)
+ * [Custom Boxes](#adding-custom-boxes)
```
Comment (jlsherrill) [inline README.md]: and this to 'Custom vagrant boxes'

File: README.md (hunk 1 above)
Comment (jlsherrill) [inline README.md]: I think its just confusing to a new user otherwise.

**Comment by ehelms** [comment]
Updated

**Comment by jlsherrill** [comment]
ACK

**Pull request merged**